devtools-ci-index.yaml

```diff
@@ -4587,22 +4575,14 @@
             ci_project: 'devtools'
             ci_cmd: '/bin/bash cico_build_release.sh'
             timeout: '2h'
-        - '{ci_project}-{git_repo}-che-nightly-test-master-temporary':
+        - '{ci_project}-{git_repo}-che-nightly-test-master-happy-path':
```

Comment:
"che" in name here is redundant IMO (`{git_repo}` is "che", so the final name would be `devtools-che-che-nightly-tests-master-happy-path`.
I would prefer bit more simple name. For example `devtools-che-happy-path-nightly`.